src/components/sbb-tag-group/sbb-tag-group.e2e.ts

```diff
@@ -1,588 +1,558 @@
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 import { waitForCondition } from '../../global/testing';
+import { assert, expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { EventSpy } from '../../global/testing/event-spy';
+import { SbbTagGroup } from './sbb-tag-group';
+import { SbbTag } from '../sbb-tag';
+import '../sbb-tag';
 
 describe('sbb-tag-group', () => {
-  let element: E2EElement, page: E2EPage;
+  let element: SbbTagGroup;
 
   describe('multiple mode', () => {
     describe('no initialized checked tag', () => {
       beforeEach(async () => {
-        page = await newE2EPage();
-        await page.setContent(`
-      <sbb-tag-group multiple>
-        <sbb-tag id="sbb-tag-1" value="tag1">Tag 1</sbb-tag>
-        <sbb-tag id="sbb-tag-2" value="tag2">Tag 2</sbb-tag>
-        <sbb-tag id="sbb-tag-3" value="tag3">Tag 3</sbb-tag>
-      </sbb-tag-group>
-    `);
-        element = await page.find('sbb-tag-group');
-
-        await page.waitForChanges();
+        element = await fixture(html`
+          <sbb-tag-group multiple>
+            <sbb-tag id="sbb-tag-1" value="tag1">Tag 1</sbb-tag>
+            <sbb-tag id="sbb-tag-2" value="tag2">Tag 2</sbb-tag>
+            <sbb-tag id="sbb-tag-3" value="tag3">Tag 3</sbb-tag>
+          </sbb-tag-group>
+        `);
       });
 
       it('renders', async () => {
-        expect(element).toHaveClass('hydrated');
+        assert.instanceOf(element, SbbTagGroup);
       });
 
       it('should have no default activated tag', async () => {
-        expect((await element.findAll('sbb-tag[checked]')).length).toBe(0);
-        expect(await element.getProperty('value')).toEqual([]);
+        expect(element.querySelectorAll('sbb-tag[checked]').length).to.be.equal(0);
+        expect(element.value).to.be.empty;
```

Comment:
I think empty and empty array is not the same?

Reply:
It should work, but I can add an additional check (`.to.be.an('array').that.is.empty;`) as suggested by the [doc](https://www.chaijs.com/api/bdd/#method_empty:~:text=a%20target%E2%80%99s%20type.-,expect(%5B%5D).to.be.an(%27array%27).that.is.empty%3B,-Add%20.not).

Reply:
good idea, otherwise I think there could also be an empty string, but we want to be sure thats an empty array